pkgs/development/tools/ocaml/ocp-build/default.nix

```diff
@@ -1,12 +1,16 @@
-{ stdenv, fetchurl, ocaml, findlib, ncurses, camlp4 }:
+{ stdenv, fetchFromGitHub, ocaml, findlib, ncurses, camlp4 }:
+
+let version = "1.99.16-beta"; in
 
 stdenv.mkDerivation {
```

Comment:
Ocp-build ships with some libs which are now installed in a standard findlib directory, and can be used by nixos if you use `buildOcaml` instead of `stdenv.mkDerivation`